.github/workflows/bcol-api-check.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       BCOL_DEBIT_ACCOUNT_VERSION: "fake"
       BCOL_LINK_CODE: "000"
 
+      BCOL_APPLIED_CHARGE_WSDL_URL: "https://appsd1.bconline.gov.bc.ca/account/services/AppliedCharge?wsdl"
```

Comment:
Is possible using mock endpoint?

Reply:
test doesn't actually hit the actual soap service, all are using mocked response. A valid wsdl is needed for the app to parse. we would need a mock soap wsdl url.